cmd/controller-manager/app/controllermanager.go

```diff
@@ -289,7 +303,7 @@ func startServiceExportController(ctx controllerscontext.Context) (enabled bool,
 		RESTMapper:                  ctx.Mgr.GetRESTMapper(),
 		InformerManager:             informermanager.GetInstance(),
 		StopChan:                    ctx.StopChan,
-		WorkerNumber:                1,
+		WorkerNumber:                ctx.Opts.ConcurrentServiceExportReconciles,
```

Comment:
We can enlarge the work number(such as to `3`), and postpone the flag thing until there is really a need.